README.md

```diff
@@ -1,11 +1,283 @@
+* Closed Captioning/SAP Settings button
+* Picture-in-Picture (PiP) button
+* AirPlay button
+* 4 app-custom sidebar buttons
+
+This video controls implementation allows for a few runtime customizations, that you can set on a player by player basis. This includes:
+* Setting the tint color for the controls (e.g., matching your app’s brand)
+* Hiding various elements of the controls (useful for smaller view versus full-screen playback)
+* Setting any of the 4 app-custom sidebar buttons
+
+The built-in tint color of the default video player controls UX is <span style="color:magenta">pink/magenta</span>. This is deliberate; feel free to keep it or change it to mesh with your app’s design or brand theme. The built-in tint color of the ad’s UX is <span style="color:gold">yellow/gold</span>. This cannot be changed at this time. Because of this, we advise that you don’t tint your main controls yellow as well. You also don’t want to use black or gray, because of video contrast, your controls will not be very visible. White, on the other hand is fair game. There is a slightly darkened canvas that is layered above the video, but below the controls layer, that helps make the controls pop out more.
+
+The player controls are shown under several well-established circumstances. This includes whenever the video is paused, such as before a video starts (with AutoPlay off) or while buffering, after any video finishes (with AutoPlay off), or after all videos linked to the player finish. They also will display (fade in) on demand whenever a video is tapped. If a video is actively playing, the controls will automatically hide (fade out) after a predetermined number of seconds. At any time the controls are shown, they can be quickly hidden by tapping on the video (not over a button, of course).
+
+The default player controls UX implementation includes 4 optional app-specific sidebar buttons. You can set any or all of these to use as you see fit. This was built to allow for app-specific video overlay customization in anticipation for up to 4 new behaviors. Because these 4 sidebar buttons are built right into the default controls UX, they will automatically fade in/out with the rest of the video controls. There is no need to handle any of that logic or attempt to synchronize to the animation timings.
+
+The complete implementation of the default player controls UX is open-source and provided to serve as an implementation sample of its own. Feel free to inspect it, copy it, modify it at will.
+
+The default iOS Custom Controls UX implementation repo can be found here: 
+[O2 Mobile SDK Controls for iOS](https://github.com/aol-public/OneMobileSDK-controls-ios)
+
+## TL;DR – Quick Start <a name="qs"></a>
+Want to dive right in, quickly and directly, you can jump here to get started using our legacy documentation: 
+[Getting Started for iOS](https://github.com/aol-public/OneMobileSDK-playground-ios/blob/master/Usage%20guide.md)
```

Comment:
It is not legacy - it is actual.

Reply:
you're right ... this goes away once this doc is installed; I wanted them to possibly live side-by-side for some time, but we didn't put this out yet, so we should ditch it.  Do you think the old non-legacy doc should be renamed and left along side?